tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "@tsconfig/node20/tsconfig.json",
+  "include": [
+    "src/**/*"
+  ],
+  "exclude": [
+    "node_modules",
+    "**/*.spec.ts"
+  ],
+  "compilerOptions": {
+    "esModuleInterop": true,
+    "outDir": "lib",
+    "declaration": true,
+    "declarationMap": true,
+    "lib": ["ES2023"]
```

Comment:
For my own curiosity, what's this for? I read the docs for this, but I don't understand. 😅

Reply:
@alexplischke are you referring to your suggestion to add `"types": ["cypress"]`? I was asking about `"lib": ["ES2023"]`. We posted nearly simulataneously so my question was maybe unclear.

Reply:
hmm..  I can answer this weird issue. Yesterday somehow I got an error
```
'replaceAll' does not exist on type 'string'
```
After some searches I added this line. While today I tried with including and excluding this line, no error got shown. I don't know why but I'll remove it.